tests/cli/demultiplex/test_verify_syncing.py

```diff
@@ -110,3 +124,203 @@ def assert_manifest_file_contains_all_files(
+        args=["--source-directory", source_directory.as_posix()],
+        obj=cg_config_object,
+    )
+    # THEN the command should exit successfully
+    assert result.exit_code == 0
+
+    # THEN a copy complete file should be created
+    assert all(
+        Path(
+            target_directory,
+            get_nanopore_flow_cell_relative_path(flow_cell_directory),
+            DemultiplexingDirsAndFiles.COPY_COMPLETE,
+        ).exists()
+        for flow_cell_directory in flow_cell_directories
+    )
+
+    # THEN a trigger file should be created
+    assert all(
+        Path(
+            target_directory,
+            NanoporeDirsAndFiles.SYSTEMD_TRIGGER_DIRECTORY,
+            flow_cell_directory.name,
+        ).exists()
+        for flow_cell_directory in flow_cell_directories
+    )
+
+
+def test_confirm_transfer_of_nanopore_flow_cell_missing_files(
+    cg_config_object, cli_runner, nanopore_flow_cells_dir: Path, tmp_path: Path
+):
+    # GIVEN a Nanopore run with two samples
+    experiment_directories: list[Path] = get_directories_in_path(nanopore_flow_cells_dir)
+    flow_cell_directories: list[Path] = get_nanopore_flow_cell_directories(nanopore_flow_cells_dir)
+    source_directory, target_directory = set_up_source_and_target_directories(
+        folders_to_include=experiment_directories,
+        temporary_path=tmp_path,
+    )
+    cg_config_object.nanopore_data_directory = target_directory.as_posix()
+    Path(target_directory, NanoporeDirsAndFiles.SYSTEMD_TRIGGER_DIRECTORY).mkdir()
+    for flow_cell_directory in get_nanopore_flow_cell_directories(source_directory):
+        # GIVEN that a file is missing
+        list(
+            Path(target_directory, get_nanopore_flow_cell_relative_path(flow_cell_directory)).glob(
+                "*txt"
+            )
+        )[0].unlink()
+        # GIVEN that they each have manifest files
+        create_manifest_file(flow_cell_directory)
+
+    # WHEN checking if all files are present
+    result = cli_runner.invoke(
+        cli=confirm_transfer_of_nanopore_flow_cell,
+        args=["--source-directory", source_directory.as_posix()],
+        obj=cg_config_object,
+    )
+
+    # THEN the command should exit successfully
+    assert result.exit_code == 0
+
+    # THEN no copy complete file should be created
+    assert all(
+        not Path(
+            target_directory,
+            flow_cell_directory.parent.name,
+            flow_cell_directory.name,
+            DemultiplexingDirsAndFiles.COPY_COMPLETE,
+        ).exists()
+        for flow_cell_directory in flow_cell_directories
+    )
+
+    # THEN no trigger file should be created
+    assert all(
+        not Path(
+            target_directory,
+            get_nanopore_flow_cell_relative_path(flow_cell_directory),
+            DemultiplexingDirsAndFiles.COPY_COMPLETE,
+        ).exists()
+        for flow_cell_directory in flow_cell_directories
+    )
+
+
+def set_up_source_and_target_directories(
```

Comment:
maybe a docstring giving context for the use of this function would be good.

Reply:
Added 👍